modules/widgets/goodreads.php

```diff
@@ -95,7 +95,7 @@ function goodreads_user_id_exists( $user_id ) {
 		$response = wp_remote_head(
 			$url, array(
 				'httpversion' => '1.1',
-				'timeout'     => 3,
+				'timeout'     => 10,
```

Comment:
Any specific reason why you picked `10`, and not a higher or lower number?

Reply:
IIRC it worked with something like 6 or 7 but I decided to go with 10 (which was also suggested in the jpop-issue) because I think it allows room for other cases which need a higher timeout. 

In my head, I'm trying to apply the same logic as dynamic arrays. When the array is full, it's better to double the size to avoid frequent changes.